### PR TITLE
KH-400: Configure drugs to be OpenMRS drugs (limited to NCD specific list)

### PR DESCRIPTION
**Pull request by kazlaw**
Link to ticket [here](https://mekomsolutions.atlassian.net/browse/KH-400)

### CODE REVIEW

**Comment by kazlaw** [comment]
AC. Drug list configured

**Comment by kazlaw** [comment]
Drug list configured

[Screencast from 21-11-23 10:30:00.webm](https://github.com/openmrs/ozone-distro-cambodia/assets/36139599/9c392b5e-64a8-4d5e-b896-725d4f2d46dd)

[Screencast from 21-11-23 10:50:38.webm](https://github.com/openmrs/ozone-distro-cambodia/assets/36139599/09cbf15a-ca59-40f4-b5ab-5ea4f02482b7)

**Comment by kazlaw** [comment]
Able to order the drugs from order basket

[Screencast from 21-11-23 11:02:29.webm](https://github.com/openmrs/ozone-distro-cambodia/assets/36139599/45ca2254-ed6f-4faa-a2fe-b539b425b98a)

**Comment by kdaud** [comment]
@kazlaw there are some files with conflicts that need to be resolved.

File: configs/openmrs_config/ampathforms/NCD_Screening.json [hunk 1]
```diff
@@ -1187,6 +1187,20 @@
...
+                                "buttonType": "ghost",
+                                "buttonLabel": "Add +"
+                            },
+                            "id": "orderDrugs",
+                            "label": "Order for Drugs here:",
+                            "hide": {
+                                "hideWhenExpression": "isEmpty(medicinePrescribed) || medicinePrescribed !== '1065AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA'"
+                            }
+                        }
                         }
```
Comment (kdaud) [inline configs/openmrs_config/ampathforms/NCD_Screening.json:1204]: @kazlaw could you take a look at this curl bracket looks like has an issue.

**Pull request merged**